ruby-monday-blog/app/models/post.rb

```diff
@@ -15,34 +15,34 @@
 #
 
 class Post < ActiveRecord::Base
-
-  has_attached_file :image, styles: {:thumb => "100x100>"}
-
-  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
-
+  # associations
   has_many :post_tags, inverse_of: :post
   has_many :tags, through: :post_tags
-
+  has_attached_file :image, styles: { thumb: '100x100>' }
   belongs_to :author, class_name: 'User'
+  accepts_nested_attributes_for :tags
+
+  before_validation :autosave_associated_records_for_tags
 
+  # validations
+  validates_attachment_content_type :image, content_type: /\Aimage\/.*\Z/
   validates :body, presence: true
   validates :title, presence: true
 
-  accepts_nested_attributes_for :tags
-
+  # scopes
   scope :ordered_by_created_at, -> { order(created_at: :desc) }
 
+  # gems
   paginates_per 10
 
-  def autosave_associated_records_for_tags
-    tags.each { |tag| self.tags << prepare_tag(tag) }
-  end
+  # delegations
+  delegate :full_name, to: :author, prefix: true
 
-  def author_full_name
```

Comment:
I think Allan moved it to the User model (https://github.com/code-newbies/ruby-monday-blog/pull/80/files#diff-aed553004de36aa9a2ccb05f2ac9ef4fR41).